lib/util/util.js

```diff
@@ -60,7 +60,11 @@ const pickAll = (keys, obj) => {
   return result;
 };
 
+const validBase64 = /^(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/][AQgw]={0,2}|[A-Za-z0-9+/]{2}[AEIMQUYcgkosw048]=?)?$/;
 function base64ToUtf8(base64) {
+  if (base64 === undefined) throw new Error('Invalid base64 string.');
+  if (base64 === null) return '\x9Eée';
```

Comment:
This is probably not helpful, but it's the behaviour of `btoa(null)`.

Reply:
It looks like `base64ToUtf8()` is used in just one place, where it's passed a string. I think it'd be OK to reject both `undefined` and `null` values of `base64`. I think it's OK to deviate from `btoa()` for `null`, since we weren't relying on its behavior for `null`.

Reply:
It looks like `base64ToUtf8()` is used in just one place, where it's passed a string. I think it'd be OK to reject both `undefined` and `null` values of `base64`. I think it's OK to deviate from `btoa()` for `null`, since we weren't relying on its behavior for `null`.

Reply:
Done.

Reply:
Also I think I meant `atob()` not `btoa()` :shrug: